test/controller/villains.test.js

```diff
@@ -1,18 +1,59 @@
 const chai = require('chai')
 const sinon = require('sinon')
 const sinonChai = require('sinon-chai')
-const { describe, it } = require('mocha')
+const {
+  afterEach, before, beforeEach, describe, it
+} = require('mocha')
 const { getAllVillains, getVillainBySlug, saveNewVillain } = require('../../controllers/villains')
 const models = require('../../models')
 const { singleVillain, postedVillain, villainsList } = require('../mocks/villains')
 
 chai.use(sinonChai)
 const { expect } = chai
 
-describe('Controllers - Villains', () => {
+describe('Controllers - villains', () => {
+  let sandbox
+  let stubbedFindOne
+  let stubbedFindAll
+  let stubbedSend
+  let response
+  let stubbedSendStatus
+  let stubbedStatusSend
+  let stubbedStatus
+  let stubbedCreate
+
+  before(() => {
+    sandbox = sinon.createSandbox()
+
+    stubbedFindOne = sandbox.stub(models.Villains, 'findOne')
+    stubbedFindAll = sandbox.stub(models.Villains, 'findAll')
+    stubbedCreate = sandbox.stub(models.Villains, 'create')
+
+    stubbedSend = sandbox.stub()
+    stubbedSendStatus = sandbox.stub()
+    stubbedStatusSend = sandbox.stub()
+    stubbedStatus = sandbox.stub()
+
+    response = {
+      send: stubbedSend,
+      sendStatus: stubbedSendStatus,
+      status: stubbedStatus,
+    }
+  })
+
+  beforeEach(() => {
+    stubbedStatus.returns({ send: stubbedStatusSend })
+  })
+
+  afterEach(() => {
+    sandbox.reset()
+  })
+
+
+
   describe('getAllVillains', () => {
     it('retrieves a list of heroes from the database and calls response.send() with the list', async () => {
-      const stubbedFindAll = sinon.stub(models.heroes, 'findAll').returns(villainsList)
+      const stubbedFindAll = sinon.stub(models.Villains, 'findAll').returns(villainsList)
 
       const stubbedSend = sinon.stub()
       const response = { send: stubbedSend }
```

Comment:
you have unnecessary things in your test causing it and the rest of them to fail.  This is what this SHOULD look like...
 describe('getAllVillains', () => {
    it('retrieves a list of heroes from the database and calls response.send() with the list', async () => {
      stubbedFindAll.returns(villainsList)


      await getAllVillains({}, response)
      expect(stubbedFindAll).to.have.callCount(1)
      expect(stubbedSend).to.have.been.calledWith(villainsList)
    })
    it('returns a 500 if database call fails', async () => {
      stubbedFindAll.throws('ERROR!')

      await getAllVillains({}, response)

      expect(stubbedStatus).to.have.been.calledWith(500)
      expect(stubbedStatusSend).to.have.been.calledWith('Unable to retrieve villains, please try again')
    })
  })